src/main/resources/lang/zh_cn/messages.yml

```
@@ -663,3 +663,15 @@ CRASH_MANAGER_CRASH_RECOVERY_ALERT_DESCRIPTION: |
   
   对此带来的不便，我们深表歉意。
 BOOT_TIME: "PeerBanHelper 启动完成，用时 {}ms"
+ECOMODE_FAILED: "Windows EcoQos 效率模式可能与您的平台不兼容：{}，操作已取消"
+JBR_REQUIRED_TITLE: "Java 运行时环境不兼容"
+JBR_REQUIRED_DESCRIPTION: |
+  PeerBanHelper 在 Windows 平台下需要 JetBrains Runtime (JBR) 才能正常运行。  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)